applications/crossbar/src/modules/cb_registrations.erl

```diff
@@ -198,36 +199,24 @@ merge_response(JObj, Regs) ->
                         end
                 end, Regs, kz_json:get_value(<<"Fields">>, JObj, [])).
 
+-spec maybe_default_port(integer(), nklib:scheme(), dict:dict()) -> integer().
+maybe_default_port(0, 'sips', _) -> 5061;
+maybe_default_port(0, 'sip', <<"TLS">>) -> 5061;
+maybe_default_port(0, 'sip', <<"tls">>) -> 5061;
+maybe_default_port(0, 'sip', _) -> 5060;
```

Comment:
@lazedo how strings like <<"tls">> may wrote here as 'tls'
If i write 'tls' then functions is not works. Thinks it is related to type casting. But not know what is type must be used.